jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/extensions/WebSocketExtensionFactory.java

```diff
@@ -47,6 +49,7 @@
 
     public WebSocketExtensionFactory(WebSocketContainerScope container)
     {
+        lifecycle = new ContainerLifeCycle();
```

Comment:
probably need to extend the `toString()` method to call the factories `toString` so that dump is meaningful